plugins/inputs/smart/README.md

```diff
@@ -0,0 +1,103 @@
+# Telegraf S.M.A.R.T. plugin
+
+Get metrics using the command line utility `smartctl` for S.M.A.R.T. (Self-Monitoring, Analysis and Reporting Technology) storage devices. SMART is a monitoring system included in computer hard disk drives (HDDs) and solid-state drives (SSDs)[1] that detects and reports on various indicators of drive reliability, with the intent of enabling the anticipation of hardware failures.
+See smartmontools (https://www.smartmontools.org/).
+
+If no devices are specified, the plugin will scan for SMART devices via the following command:
+
+```
+smartctl --scan
+```
+
+Metrics will be reported from the following `smartctl` command:
+
+```
+smartctl --info --attributes --health -n <nocheck> --format=brief <device>
```

Comment:
This is probably the most important thing to figure out.  Did the format change or does this drive just not have any attributes?